### PR TITLE
Added jackson

### PR DESCRIPTION
**Pull request by nulls**

### CODE REVIEW

**Review by sanyavertolet — APPROVED** [review]

**Comment by sanyavertolet** [comment]
No diktat yet?

File: src/commonMain/kotlin/com/saveourtool/osv4k/OsvSchema.kt [hunk 1]
```diff
@@ -24,12 +24,50 @@ data class OsvSchema<D, A_D, A_E, A_R_D> (
 
     val id: String,
     @Serializable(with = LocalDateTimeRfc3339Serializer::class)
+    @JsonSerialize(
+        using = LocalDateTimeRfc3339JacksonSerializer::class,
+        contentUsing = JsonSerializerNone::class,
```
Comment (nulls) [inline src/commonMain/kotlin/com/saveourtool/osv4k/OsvSchema.kt:29]: according to https://youtrack.jetbrains.com/issue/KT-22704: the default values in annotations should work as default values in actual type -- but it doesn't

**Comment by nulls** [comment]
> No diktat yet?

not yet, I even didn't configure any github action

File: src/jvmTest/kotlin/com/saveourtool/osv4k/OsvSchemaJacksonTestUtil.kt [hunk 2]
```diff
@@ -11,13 +15,17 @@ object OsvSchemaJacksonTestUtil {
     private val objectMapper = ObjectMapper()
         .registerModule(JavaTimeModule())
         .registerKotlinModule()
+        .setSerializationInclusion(JsonInclude.Include.NON_NULL)
```
Comment (nulls) [inline src/jvmTest/kotlin/com/saveourtool/osv4k/OsvSchemaJacksonTestUtil.kt:18]: should be moved to OsvSchema

File: src/jvmTest/kotlin/com/saveourtool/osv4k/OsvSchemaJacksonTestUtil.kt [hunk 3]
```diff
@@ -11,13 +15,17 @@ object OsvSchemaJacksonTestUtil {
...
+
+    private val prettyWriter = objectMapper.writerWithDefaultPrettyPrinter()
+        .with(DefaultPrettyPrinter().withoutSpacesInObjectEntries())
```
Comment (nulls) [inline src/jvmTest/kotlin/com/saveourtool/osv4k/OsvSchemaJacksonTestUtil.kt:21]: it doesn't help -- test is failed.

**Review by kgevorkyan — APPROVED** [review]

File: src/jvmTest/kotlin/com/saveourtool/osv4k/OsvSchemaJacksonTestUtil.kt (hunk 2 above)
Comment (nulls) [reply src/jvmTest/kotlin/com/saveourtool/osv4k/OsvSchemaJacksonTestUtil.kt]: done

**Pull request merged**